packages/zoe/src/zoeService/zoe.js

```diff
@@ -335,7 +332,7 @@ function makeZoe(vatAdminSvc, zcfBundleName = undefined) {
             initialAllocation,
             instanceAdmin,
             proposal,
-            brandToPurse,
+            withdrawPayments,
```

Comment:
I'll just mention that this aspect of the refactoring is also a POLA improvement. `withdrawPayments` is much less powerful than `brandToPurse`. If the callee is buggy, it can now do less damage than before.